samples/dynamic-headers/custom-visual/capabilities.json

```diff
@@ -0,0 +1,44 @@
+{
+  "dataRoles": [
+    {
+      "displayName": "Static title columns",
+      "name": "static_title_columns",
+      "kind": "GroupingOrMeasure"
+    },
+    {
+      "displayName": "Dynamic header titles",
```

Comment:
Consider renaming the data role to "Dynamic column headers". Propagate this change accordingly through all the implementation for consistency.